infra/helper.py

```diff
@@ -713,6 +715,26 @@ def build_fuzzers_impl(  # pylint: disable=too-many-arguments,too-many-locals,to
 
 def build_fuzzers(args):
   """Builds fuzzers."""
+  if args.engine == 'centipede':
+    # Centipede always requires separate binaries for sanitizers:
+    # An unsanitized binary, which Centipede requires for fuzzing.
+    # A sanitized binary, placed in the child directory.
+    sanitized_binary_directories = (
+        ('none', ''),
+        (args.sanitizer, f'{args.project.name}_{args.sanitizer}'),
+    )
+    return all([
```

Comment:
nit: remove the [ ] parts here to not make this eagerly evaluated, so that we early exit on the first failure.

`all([f(x) for x in ...])` will evaluate all arguments before the `all()` part is run.
`all(f(x) for x in ...)` will stop at the first false.